vizro-core/docs/pages/user-guides/install.md

```diff
@@ -82,12 +80,10 @@ You should see a return output of the form `x.y.z`.
 
 ## Upgrade
 
+Check the [release notes](https://github.com/mckinsey/vizro/blob/main/vizro-core/CHANGELOG.md) for any notable breaking changes before migrating an existing project.
+
 To change the version of Vizro installed:
 
```

Comment:
Same comment if we need this doc section at all?

Reply:
As above. I think it's worth retaining.